the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
+
 		<!-- json-simple -->
-		
+
 		<dependency>
```

Comment:
MultipleWindowsPage:
 Should click on the link based on link text instead of index.